src/objects/sap/zcl_abapgit_sap_namespace.clas.abap

```diff
@@ -26,4 +26,31 @@ CLASS ZCL_ABAPGIT_SAP_NAMESPACE IMPLEMENTATION.
     SELECT SINGLE editflag FROM trnspace INTO lv_editflag WHERE namespace = iv_namespace.
     rv_yes = boolc( sy-subrc = 0 AND lv_editflag = 'X' ).
   ENDMETHOD.
+
+
+  METHOD zif_abapgit_sap_namespace~split_by_name.
+    DATA lv_regex TYPE string.
```

Comment:
so it shows up when searching for the old function module,

```suggestion
* use this method instead of function module RS_NAME_SPLIT_NAMESPACE
    DATA lv_regex TYPE string.
```